src/compressed_tensors/quantization/quant_args.py

```diff
@@ -250,6 +245,10 @@ def pytorch_dtype(self) -> torch.dtype:
         else:
             raise ValueError(f"Invalid quantization type {self.type}")
 
+    @deprecated("QuantizationArgs.observer")
+    def get_observer(self) -> str:
+        return self.observer
+
```

Comment:
Is there a reason to move the location?

Reply:
Deprecated definitions should be moved to the bottom, as they're less relevant for a reader

Reply:
Same logic behind private functions being towards the bottom